src/components/Toast/README.md

```diff
@@ -91,23 +99,31 @@ Messages should be:
 - Product updated
 - Collection added
 - Customer updated
-- No internet connection
+- Internet disconnected
+- Connection timed out
 
 #### Don’t
 
+- No internet connection
 - Your product has been successfully updated
 - We were unable to save the customer
 - Your Order was Archived Today
 - Discount: Saved successfully
 
 <!-- end -->
 
-### Action
+### Toasts with actions
+
+Only include actions in toasts if the same action is available elsewhere on the page. For example:
+
+- If merchants need to reload a section, offer the call to action [Reload] in the toast message. If they miss the toast, they can also refresh the entire page.
+- If merchants delete an image, offer the option to [Undo] the deletion. If they miss it in the toast, they can still retrieve it from {x}.
 
 Action should:
 
-- Keep the action label short. Preferably 1 verb
-- Not have actions for dismissing toast
+- Keep the action label short, preferably 1 verb.
+- Not have actions, like [Cancel], for dismissing toast. The [X] to dismiss is already included in the component.
+- Be used with a duration of 10000 milliseconds for accessibility.
```

Comment:
@dpersing Is this where this belongs? I was thinking this was good enough for now until there is an accessibility section.